app/models/manageiq/providers/kubevirt/inventory/collector.rb

```diff
@@ -27,7 +27,7 @@ def initialize_for_targeted_refresh
     name = @target.name
     @nodes = {}
 
-    @manager.with_provider_connection do |connection|
+    @manager.with_provider_connection(:namespace => @target.location) do |connection|
       if @target.template?
```

Comment:
Totally unrelated to this change, I don't think this can ever happen based on `if refresh_target.kind_of?(ManageIQ::Providers::Kubevirt::InfraManager::Vm)` above.  That wouldn't include `ManageIQ::Providers::Kubevirt::InfraManager::Template` instances